run.sh

```diff
@@ -4,4 +4,5 @@ for filename in bin/*
 do
     metricName=$(basename $filename | cut -f 1 -d '.' | sed -r 's/(^|_|-)([a-z])/\U\2/g')
     aws cloudwatch put-metric-data --metric-name $metricName --namespace $namespace --value $(node $filename) --unit Milliseconds --timestamp $(date -u +"%Y-%m-%dT%H:%M:%SZ") --region us-east-1
+    sleep 10
```

Comment:
The speed of running all the tests will increase with every test we add. Is this need for _all_ tests or only a selected few? 
This also means the tests will accumulate in the CI once they go over the 1 minute window before the next test kicks in.

Reply:
It's needed for all publish tests running in same space. Alternatively we can use different space for every test. Do we plan to add more tests anytime soon ?

Reply:
> Do we plan to add more tests anytime soon ?
I don't think so. This is an implicit config though and very easy to miss if you are adding new tests without knowing this.

> It's needed for all publish tests running in same space. Alternatively we can use different space for every test
Is the debounce done on a space level?

Reply:
> Is the debounce done on a space level?

Space & environment

> I don't think so. This is an implicit config though and very easy to miss if you are
adding new tests without knowing this.

It might be a good time to create a readme file and document the internals of the repo.